### PR TITLE
Lesson 7 Task 4 - Text Length

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Class TextLength created with method to calculate the text lenght
2. Test class created within parameterizedtest and provided sample texts

### CODE REVIEW

File: src/main/java/lesson7/task4/TextLength.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package lesson7.task4;
+
+public class TextLength {
```
Comment (strzeleckik) [inline src/main/java/lesson7/task4/TextLength.java]: TextUtils maybe?

File: src/main/java/lesson7/task4/TextLength.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+    public static int calculateTextLength(String text) {
+        int textLength = text.length();
```
Comment (strzeleckik) [inline src/main/java/lesson7/task4/TextLength.java]: what if text is null? :)

**Review by strzeleckik — CHANGES_REQUESTED** [review]
feedback to consider

**Comment by jean-farinea** [comment]
Changes done, please review.

File: src/test/java/lesson7/task4/TextUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    @ParameterizedTest
+    @MethodSource("provideSampleTexts")
+    void shouldCheckCalculateTextLength(String text) {
+        //given
+        int expectedLength = text.length();
+        //when
+        int calculatedLength = TextUtils.calculateTextLength(text);
+        //then
+        Assertions.assertEquals(expectedLength, calculatedLength);
```
Comment (strzeleckik) [inline src/test/java/lesson7/task4/TextUtilsTest.java:21]: bad practice to check positive and negative scenario in one test

**Review by strzeleckik — CHANGES_REQUESTED** [review]
feedback in comment

**Comment by jean-farinea** [comment]
Test adjusted, please review.

**Review by strzeleckik — APPROVED** [review]
ok

**Pull request merged**